modules/matterhorn-admin-ui-ng/src/main/webapp/scripts/modules/events/controllers/eventController.js

```diff
@@ -149,11 +149,16 @@ angular.module('adminNg.controllers')
                         if (el.length > 0) {
                             if (el.is('[type=checkbox]') || el.is('[type=radio]')) {
                                 if (value === 'true' || value === true) {
-                                    el.attr('checked','checked');
+                                    el.attr('checked', true);
```

Comment:
[attr](http://api.jquery.com/attr/#attr2) should just set the value of the attribute and a checked attribute should afaik always set the element to checked regardless of its value. At least according to the specs…

Thus, it would make sense to remove the attribute instead like Opencast already does in recent versions:

https://github.com/opencast/opencast/blob/2d383df990cd8810876b4d6f1f1ec50b60fd16d9/modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/eventController.js#L158-L162

Reply:
I'm not sure I follow. Yes setting el.attr('foo',value); works in the main, but for checkboxes and radio button, to unset them (ie the value=false) either you have set value to null or call el.removeAttr('checked');  explictly

` if (value !== 'true' && value !== true ) {
    value = null; 
 }
 el.attr('checked', value)
`

would be an alternate way of writing it I guess?

Reply:
@JamesUoM The attribute "checked" needs to be set to the value "checked" (see https://www.w3schools.com/tags/att_input_checked.asp). There has been recently a fix the removes the attribute "checked" correctly: https://github.com/opencast/opencast/commit/eb7754f25b0d31a8a40eb0e91fffdc16e5f94935#diff-2ed317613602203533b1e42388734725

So the code @lkiesow should be correct but is not yet in 4.x

Reply:
Sorry, I was confused I thought I was looking at my code the 6.x's. Yes I've remove the attr.